source/wp-content/themes/wporg-developer-2023/theme.json

```diff
@@ -3,37 +3,79 @@
 	"version": 2,
 	"settings": {
 		"blocks": {
-            "wporg/code-reference-title": {
+			"wporg/code-reference-title": {
 				"color": {
-                    "palette": [
+					"palette": [
 						{
-                            "slug": "syntax-black",
-                            "color": "#000000",
-                            "name": "Black"
-                        },
-                        {
-                            "slug": "syntax-red",
-                            "color": "#d63638",
-                            "name": "Red"
-                        },
+							"slug": "syntax-black",
+							"color": "#000000",
+							"name": "Black"
+						},
 						{
-                            "slug": "syntax-green",
-                            "color": "#008a20",
-                            "name": "Green"
-                        },
+							"slug": "syntax-red",
+							"color": "#d63638",
+							"name": "Red"
+						},
 						{
-                            "slug": "syntax-blue",
-                            "color": "#135e96",
-                            "name": "Blue"
-                        },
+							"slug": "syntax-green",
+							"color": "#008a20",
+							"name": "Green"
+						},
 						{
-                            "slug": "syntax-grey",
-                            "color": "#646970",
-                            "name": "Grey"
-                        }
+							"slug": "syntax-blue",
+							"color": "#135e96",
+							"name": "Blue"
+						},
+						{
+							"slug": "syntax-grey",
+							"color": "#646970",
+							"name": "Grey"
+						}
 					]
 				}
 			}
 		}
+	},
+	"styles": {
+		"blocks": {
+			"wporg/chapter-list": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--small)"
+				},
+				"elements": {
+					"heading": {
+						"typography": {
+							"fontSize": "var(--wp--preset--font-size--normal) !important"
+						}
+					},
+					"link": {
+						"color": {
+							"text": "var:preset|color|blueberry-1"
+						}
+					}
+				}
+			},
+			"wporg/table-of-contents": {
+				"color": {
+					"background": "var:preset|color|blueberry-4",
+					"text": "var:preset|color|blueberry-1"
```

Comment:
Why do this use the `var:` format and the other ones don't? Is there a limitation?

Reply:
No, probably just the way I copied them - swapped in the custom properties 👍🏻